apps/angular/src/app/features/anime/components/anime-table/anime-table.component.ts

```diff
@@ -295,4 +297,9 @@ export class AnimeTableComponent implements OnInit, AfterViewInit, OnDestroy {
   public ngOnDestroy(): void {
     this.subscriptionOnChanges.unsubscribe();
   }
+
+  /** Handle quit button click. */
+  public handleQuitButton(): void {
+    this.userService.removeJwtFromLocalStorage();
+  }
```

Comment:
On Angular projects we use `on` prefix for event handlers, also your method name is missing high context ([wiki](https://wiki.saritasa.rocks/frontend/languages/ts-js/naming#ahclc-pattern))
```suggestion
  /** Handle quit button click. */
  public onQuitButtonClick(): void {
    this.userService.removeJwtFromLocalStorage();
  }
```

Reply:
Renamed